jenkins/test-history/gen_html.py

```diff
@@ -45,6 +46,38 @@
 ])
 
 
+PREFIX_TO_URL = {}
+
+def prefix_for(prefix):
+    if 'azure' in prefix:
+        new = 'azure$'
+    elif 'rktnetes' in prefix:
+        new = 'rktnetes$'
+    elif 'gs://kubernetes-jenkins/' in prefix:
+        new = ''
+    else:
+        new = prefix[5:-1].replace('/', '_') + '$'
+    PREFIX_TO_URL[new] = prefix
+    return new
+
+
+def slugify(inp):
+    inp = re.sub(r'[^\w\s-]+', '', inp)
+    return re.sub(r'\s+', '-', inp).lower()
+
+
+def gubernator_href(suite, build_number, test_name):
+    if '$' in suite:
+        prefix, suite = suite.split('$')
+        path = PREFIX_TO_URL[prefix + '$']
+    else:
+        path = PREFIX_TO_URL['']
+    if path.startswith('gs://'):
+        path = path[5:]
+    return '<a href="https://k8s-gubernator.appspot.com/build/%s%s/%s#%s">' % (
```

Comment:
I'd rather this just return the link so that the HTML structure is more clear later on.